src/modules/datePickerDropdownYear.js

```diff
@@ -87,11 +88,10 @@ class DatePickerDropdownYear extends React.PureComponent {
         const { maxDate } = this.props;
 
         if (!maxDate) {
-            const datePickerWrapElement = ReactDOM.findDOMNode(document.querySelector('.date-picker-dropdown-year-scrollbar > div'));
             const selectedElement = ReactDOM.findDOMNode(document.querySelector('.date-picker-year-option-is-selected'));
             const selectedPosistion = selectedElement.offsetTop - 11;
 
-            domUtils.scrollTo(selectedPosistion, 0, datePickerWrapElement);
+            this.scrollComponent.scrollTop(selectedPosistion);
```

Comment:
Love the ref being used here and removing the `domUtils.scrollTo()` function. It's not really all that needed to see an scrolling animation.